tests/examples_e2e/README.md

```diff
@@ -0,0 +1,73 @@
+Temporary (?) instructions how to set up your environment for E2E tests
+
+# Prerequisites
+Prepare following information:
+* billing account id
+* your organization id
+* parent folder under which resources will be created
+  * (you may want to disable / restore to default some organization policies under this folder) 
+* decide in which region you want to deploy (choose one, that has wide service coverage)
+* prepare a prefix, suffix and a timestamp for you (this is to provide project and other resources name uniqueness)
+* prepare service account that has necessary permissions (able to assign billing account to project, resource creation etc)
+
+# Option 1 - automatically provision and de-provision testing infrastructure
```

Comment:
Indeed, it might be easier to handle option 1 also through a file (and pointing to it using environment variable).

For now, I prefer to keep both options to facilitate quickly running a few tests locally. If I'll be able to trim down setup of test resources to reasonable time (1-2 minutes?) then I'm perfectly happy to remove it. It's on TODO list before merging.